index.bs

```diff
@@ -260,6 +265,9 @@ interface ImageCapture {
 
   <dt><dfn dict-member for="PhotoSettings"><code>fillLightMode</code></dfn></dt>
   <dd>This reflects the desired <a>fill light mode</a> (flash) setting.</dd>
+
+  <dt><dfn dict-member for="PhotoSettings"><code>imageFormat</code></dfn></dt>
+  <dd>When the UA returns a {{Blob}} from {{takePhoto()}}, the format and {{Blob/type}} MUST be one of the supported <a>image formats</a> enumerated in {{PhotoCapabilities/imageFormat}}, if any.</dd>
```

Comment:
These look like normative steps for the [takePhoto](https://w3c.github.io/mediacapture-image/#dom-imagecapture-takephoto) algorithm, and should probably move there.

As currently written it doesn't actually say the UA should look at the imageFormat setting at all. @yellowdoge [said](https://github.com/w3c/mediacapture-image/pull/185#discussion_r123317570) that *"takePhoto() should fail if unsupported type was requested"* - That would be one way to go. But which error? We'd need to specify that.